src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1484,6 +1485,27 @@ export class SearchView extends ViewPane {
 			.then(() => undefined, () => undefined);
 	}
 
+
+	private _updateResults() {
+		try {
+			if (this.state === SearchUIState.Idle) {
+				window.clearInterval(this._uiRefreshHandle);
```

Comment:
Is the handle still needed? I'm not sure where it's being set or used.

Reply:
good catch, it isn't used anymore